### PR TITLE
add accessibility tests

### PR DESCRIPTION
**Pull request by djwester**
add accessibility checks to make sure we are correctly following standards

### CODE REVIEW

File: d2l-rubric-criteria-group.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 		</style>
 		<d2l-table-wrapper>
-			<table class="d2l-table">
+			<table class="d2l-table" summary="This table lists criteria and criteria group names in the first column. The first row lists level names and includes scores if the rubric uses a numeric scoring method. A similar row starts any additional criteria group.">
```
Comment (alogan519) [inline d2l-rubric-criteria-group.html]: I can localize this

Comment (djwester) [reply d2l-rubric-criteria-group.html]: do you want to do that after I merge this, or should we put this fie into a different PR?

Comment (alogan519) [reply d2l-rubric-criteria-group.html]: Actually this might not be true in our case "A similar row starts any additional criteria group."

Comment (alogan519) [reply d2l-rubric-criteria-group.html]: ya after is good.

Comment (djwester) [reply d2l-rubric-criteria-group.html]: I have updated the summary to reflect the fact that the WC has each group in it's own table

**Pull request merged**